src/Core/Content/Category/SalesChannel/NavigationRoute.php

```diff
@@ -54,14 +54,26 @@ public function getDecorated(): AbstractNavigationRoute
      * @Entity("category")
      * @OA\Post(
      *      path="/navigation/{requestActiveId}/{requestRootId}",
-     *      summary="Loads all available navigations",
+     *      summary="Fetch a navigation menu",
+     *      description="This endpoint returns a categories that can be used as a page navigation. You can either return them as a tree or as a flat list. You can also control the depth of the tree.",
      *      operationId="readNavigation",
-     *      tags={"Store API", "Navigation"},
+     *      tags={"Store API", "Category"},
      *      @OA\Parameter(name="Api-Basic-Parameters"),
-     *      @OA\Parameter(name="requestActiveId", description="Active Category ID", @OA\Schema(type="string"), in="path", required=true),
-     *      @OA\Parameter(name="requestRootId", description="Root Category ID", @OA\Schema(type="string"), in="path", required=true),
+     *      @OA\Parameter(
+     *          name="requestActiveId",
+     *          description="The active category id",
+     *          @OA\Schema(type="string", format="uuid"),
+     *          in="path",
+     *          required=true
+     *      ),
+     *      @OA\Parameter(
+     *          name="requestRootId",
+     *          description="The root category id.",
+     *          @OA\Schema(type="string", format="uuid"),
+     *          in="path",
+     *          required=true
+     *      ),
```

Comment:
There is one parameter missing - `depth`.

From an older version:

---

## Parameters

### Header

`sw-include-seo-urls` | boolean | optional

Fetch routes for every category. Default is false.

### Path

`activeId` | uuid | required

Id of the active category in the navigation tree (if not used, just set to the same as rootId .

`rootId` | uuid | required

Id of the root category for your desired navigation tree. You can use it to fetch sub-trees of your navigation tree.

> Instead of passing uuids, you can also use one of the following aliases for the activeId and rootId parameters to get the respective navigations of your sales channel.
>
> * main-navigation
> * service-navigation
> * footer-navigation
> ```POST /navigation/main-navigation/main-navigation```

### Body

`buildTree` | boolean | optional

Return the categories as a tree or as a flat list. Default is true.

`depth` | integer | optional

Determines the depth of fetched navigation levels.  Default is 2.